spec/features/ncr/work_orders/requester_edit_spec.rb

```diff
@@ -141,14 +141,6 @@
     expect(page).to have_content("delegate@example.com")
   end
 
-  scenario "has 'Discard Changes' link" do
-    visit edit_ncr_work_order_path(work_order)
-
-    click_link "Discard Changes"
-
-    expect(current_path).to eq(proposal_path(ncr_proposal))
-  end
-
```

Comment:
hah WOOPS! accident. Was failing so I was editing it but must have accidentally deleted- will restore!